### ISSUE TITLE
Shouldn't "value" key be changed to "object" when the property refers to another entity ? (e.g. in stat:dimension)

### ISSUE THREAD

**Issue by tfrancart** [flopezag/IoTAgent-Turtle#59]
My understanding is that this:

```
"stat:dimension": {
        "type": "Property",
        "value": [
            "urn:ngsi-ld:DimensionProperty:timePeriod",
            "urn:ngsi-ld:DimensionProperty:refArea"
        ]
    }
```

Should be turned to:

```
"stat:dimension": {
        "type": "Property",
        "object": [
            "urn:ngsi-ld:DimensionProperty:timePeriod",
            "urn:ngsi-ld:DimensionProperty:refArea"
        ]
    }
```

**Comment by flopezag** [comment]
Yes absolutely, we are also identified the issue and the new version of the component will include this change.

**Comment by flopezag** [comment]
Resolved in the issue #89
